config.yaml

```diff
@@ -50,7 +50,8 @@ tektonMapper:
     # if you run horizon on local machine, you need to set this to the absolute path of your kubeconfig
     # if you are running it on mac, the path may be looked like '/Users/xxx/.kube/config'
     kubeconfig: "/Users/xxx/.kube/config"
-    s3:
+    logStorage:
```

Comment:
Add some comments? Please point out what kinds of value could be taken here and what's the use case of each.